storagemarket/events.go

```diff
@@ -265,6 +265,7 @@ const (
 	ProviderEventRestart
 
 	// ProviderEventDataTransferRestartFailed means a data transfer that was restarted by the provider failed
+	// Deprecated: this event is no longer used
```

Comment:
Just want to flag a thing for the future: at some point we are going to want to re-arrange these events and clean up deprecated ones.

When this happens, two things will be needed:
- a migration - see our whole migrations process using go-ds-migrations
- a protocol upgrade - meaning that we will need to define a new version on the protocol, and translate even numbers back and forth for older protocol versions

Reply:
NOTE: this is the markets libp2p protocol here FWIW